.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: Identify and close stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * 0"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Given that this action won't work "out of the box" until/unless a `GITHUB_TOKEN` secret is configured for the repository using the template, I'm wondering if this is worth explicitly calling out in the `oss-template` README whenever we add a "How to Use this Template" section. And then we can defer to the official [Stale action](https://github.com/actions/stale) documentation for the remaining configuration.

Reply:
Pretty sure it will just work out of the box. I did not have to configure anything special for either my account or the repository on which I used this action.

https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/disabling-or-limiting-github-actions-for-a-repository#setting-the-permissions-of-the-github_token-for-your-repository

Reply:
Confirming... https://github.com/bostonion/onions

Reply:
https://github.com/bostonion/onions/actions

Neither the account nor the repo have any API keys or the like. Appears to "just work".

Reply:
confirming what John said @lelia that token is there for free https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

Reply:
ty both for confirming!